Makefile

```diff
@@ -27,10 +27,11 @@ cover:
   find tests -type d | xargs nosetests --with-coverage --cover-inclusive --cover-erase --cover-package=twilio
 
 docs-install:
-	. venv/bin/activate; pip install pdoc
+	. venv/bin/activate; pip install twilio sphinx
```

Comment:
does this install twilio in the virtualenv? is that necessary?

Reply:
I've imported `__version__` from the twilio module on `docs/config.py` . Is there a way to grab the current API version that doesn't require installing twilio?

Reply:
You don't want the current API version -- you want the next API version (the one you're building the docs for). You already have the source for that in the virtualenv.